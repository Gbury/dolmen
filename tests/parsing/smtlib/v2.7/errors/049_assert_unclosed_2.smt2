(set-logic ALL)
(assert true _)
