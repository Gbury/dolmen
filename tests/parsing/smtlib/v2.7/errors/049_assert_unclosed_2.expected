File "tests/parsing/smtlib/v2.7/errors/049_assert_unclosed_2.smt2", line 2, character 13-14:
2 | (assert true _)
                 ^
Error while parsing an assertion, read the reserved word '_',
      but expected a closing parenthesis.
